examples/mysql-kopia-repository-server/README.md

```diff
@@ -0,0 +1,430 @@
+
+```bash
+$ openssl req -newkey rsa:2048 -nodes -keyout key.pem -x509 -days 365 -out certificate.pem
+
+$ kubectl create secret tls repository-server-tls-cert --cert=certificate.pem --key=key.pem -n kanister
+```
+
+- Create Location Secrets for Kopia Repository
+
+```bash
+# The following file s3_location_creds.yaml is a sample file for creating s3 credentials secrets. It contains the credentials for accessing the s3 bucket.
+$ vi s3_location_creds.yaml
+
+apiVersion: v1
+kind: Secret
+metadata:
+   name: s3-creds
+   namespace: kanister
+   labels:
+      repo.kanister.io/target-namespace: monitoring
+type: secrets.kanister.io/aws
+data:
+   # required: base64 encoded value for key with proper permissions for the bucket
+   aws_access_key_id: <base64_encoded_aws_access_key>
+   # required: base64 encoded value for the secret corresponding to the key above
+   aws_secret_access_key: <base64_encoded_aws_secret_access_key>
+```
+
+```
+$ kubectl create -f s3_location_creds.yaml -n kanister
+```
+
+```bash
+# The following file s3_location.yaml is a sample file for creating s3 location secrets. It contains the details of the s3 bucket.
+$ vi s3_location.yaml
+
+apiVersion: v1
+kind: Secret
+metadata:
+   name: s3-location
+   namespace: kanister
+   labels:
+      repo.kanister.io/target-namespace: monitoring
+type: Opaque
+data:
+   # required: specify the type of the store
+   # supported values are s3, gcs, azure, and file-store
+   type: czM=
+   bucket: <base64_encoded_s3_bucket_name>
+   # optional: used as a sub path in the bucket for all backups
+   path: <base_64_encoded_prefix_provided_when_creating_kopia_repository>
+   # required, if supported by the provider
+   region: <base64_encoded_s3_bucket_region>
+   # optional: if set to true, do not verify SSL cert.
+   # Default, when omitted, is false
+   #skipSSLVerify: false
+   # required: if type is `file-store`
+   # optional, otherwise
+   #claimName: store-pvc
+```
+
+```
+$ kubectl create -f s3_location.yaml -n kanister
+```
+
+- Apply Secrets for Kopia Repository Server User Access, Admin Access and Repository Access
+
+```bash
+# The following command creates secrets for kopia repository server user access.
+kubectl create secret generic repository-server-user-access -n kanister --from-literal=localhost=<suitable_password_for_repository_server_user>
+
+# The following command creates secrets for kopia repository server admin access.
+kubectl create secret generic repository-admin-user -n kanister --from-literal=username=<suitable_admin_username_for_repository_server> --from-literal=password=<suitable_password_for_repository_server_admin>
+
+# The following command creates secrets for kopia repository access.
+kubectl create secret generic repo-pass -n kanister --from-literal=repo-password=<repository_password_set_while_creating_kopia_repository>
```

Comment:
Add a note that all the above secrets must not be deleted to be able to successfully recover the backups

Reply:
Sure !